### ISSUE TITLE
Custom vs elastic folder

### ISSUE THREAD

**Issue by DavidJameson** [packagesdev/packages#114]
It is unclear what is the difference between a custom folder and an elastic folder - this may be just a documentation issue but should probably be addressed somewhere.

**Comment by packagesdev** [comment]
http://s.sudre.free.fr/Software/documentation/Packages/en_2017/Package_Payload.html#2

Would an example in the table make it easier to see the difference?

**Comment by DavidJameson** [comment]
I had seen that page already — but it was completely unclear what it actually was. A custom folder can have folders inside and so they can all be moved around in one go - consequently it’s unclear to me how that’s different then an elastic folder.

An example would probably make it clear.

D

**Comment by packagesdev** [comment]
Custom folder = "my folder name"

Elastic folder = "my folder name/my subfolder name/my subsubfolder name" or "my folder name"

A custom folder represents 1 folder.

An elastic folder can represent a series of enclosed folders (that will share the same POSIX permissions and owner/group).

In 99.95% of cases, you don't need an elastic folder.

See also: http://s.sudre.free.fr/Software/Packages/Q&A_10.html

**Comment by DavidJameson** [comment]
Yes, I understand what it is — I don’t see anything “elastic” about such folder. I found that misleading. What the Q&A describes is a short cut to writing a path in a single entry to save you from creating individual folders.

Perhaps a better explanation would to be say that if you write

    a/b/c/d

it's equivalent to 

    mkdir -p a/b/c/d

and then you can (presumably) throw some content into folder ‘d’  (but not into a, b, or c)


By the way, please don’t take any of this as a COMPLAINT — on the contrary, this product has saved my ass. I wish there was a way to contribute some funds to support its developers!
